src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -98,4 +132,51 @@ private void copyUrl() {
         url.putString(USERGUIDE_URL);
         clipboard.setContent(url);
     }
+
+    /**
+     * Initializes the Command Summary which is a {@code TableView}
+     */
+    @SuppressWarnings("unchecked")
+    @FXML
+    private void initializeCommandSummary() {
+        ObservableList<Command> observableCommandList = FXCollections.observableArrayList();
+        populateCommandSummary(observableCommandList);
+
+        commandSummary.setItems(observableCommandList);
+
+        TableColumn<Command, String> actionCol = new TableColumn<>("Action");
+        actionCol.setCellValueFactory(new PropertyValueFactory<>("action"));
+
+        TableColumn<Command, String> formatExampleCol = new TableColumn<>("Format, Examples");
+        formatExampleCol.setCellValueFactory(new PropertyValueFactory<>("formatExample"));
+
+        commandSummary.getColumns().setAll(actionCol, formatExampleCol);
+    }
+
+    /**
+     * Populates the Command Summary with available commands
+     * @param commandList
+     */
+    private void populateCommandSummary(ObservableList<Command> commandList) {
+        Person samplePerson = new Person(new Nric("S4260423B"), new Name("John Doe"), new Phone("98765432"),
+                new Email("johnd@example.com"), new Address("John street"), new HashSet<>(),
+                new CallFrequency("7"));
+
+        commandList.add(new AddCommand(samplePerson));
+        commandList.add(new ClearCommand());
+        commandList.add(new DeleteCommand(Index.fromOneBased(1)));
+        commandList.add(new EditCommand(Index.fromOneBased(1),
+                new EditCommand.EditPersonDescriptor()));
+        commandList.add(new FindCommand(new NameContainsKeywordsPredicate(new ArrayList<>())));
+        commandList.add(new HistoryCommand(Index.fromOneBased(1)));
+        commandList.add(new ListCommand());
+        try {
+            commandList.add(new MarkCommand(Index.fromOneBased(1),
+                    new ContactRecord(ParserUtil.parseDate("2020-13-01"), "")));
+        } catch (ParseException e) {
+            logger.warning("HelpWindow parse error: Date format is incorrect.");
+        }
```

Comment:
```suggestion

        commandList.add(new MarkCommand(Index.fromOneBased(1),
                    new ContactRecord(LocalDate.parse("2020-13-01"), "")));

```
then don't need to have a try catch, cuz it's abit weird to have a try catch for a date that is hard coded